internal/pkg/service/common/etcdop/iterator/iterator.go

```diff
@@ -236,22 +236,23 @@ func (v *Iterator) moveToPage(resp *etcd.GetResponse) bool {
 	return true
 }
 
-func firstPageOp(prefix, end string, pageSize int, revision int64) op.GetManyOp {
-	return nextPageOp(prefix, end, pageSize, revision)
-}
-
-func nextPageOp(start, end string, pageSize int, revision int64) op.GetManyOp {
+func getPageOp(start, end string, pageSize int, revision int64, serializable bool) op.GetManyOp {
 	// Range options
 	opts := []etcd.OpOption{
 		etcd.WithFromKey(),
 		etcd.WithRange(end), // iterate to the end of the prefix
 		etcd.WithLimit(int64(pageSize)),
-		etcd.WithSort(etcd.SortByKey, etcd.SortAscend),
+		// etcd sorts results by default by the key (this is how the keys are physically stored).
+		// This sorting does not require loading all records into memory.
+		// The sort option is not needed.
+		// etcd.WithSort(etcd.SortByKey, etcd.SortAscend),
```

Comment:
Sort tu nema zmysel, je to default sort, ktory pouziva etcd = tak su data fyzicky ulozene.

Pri ostatnych sort moznostiach musi etcd nacitat vsetky zaznamy do memory, tam ich zoradit, a az tak sa aplikuje `WithLimit`. Pri default sort to nie je potrebne, a rovno z disku sa nacita iba `limit` zaznamov.

Nie som si isty ako sa etcd server zachova, ked je tu tento default sort nastaveny.
Ci je to optimalizovane, alebo sa to aj tak nacita do memory, aj ked netreba.
Tak to radsej davam prec.